src/bindings/python/src/compatibility/openvino/__init__.py

```diff
@@ -2,4 +2,61 @@
 # Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore # mypy issue #1422
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+
+# Required for Windows OS platforms
+# Note: always top-level
+try:
+    from openvino.utils import _add_openvino_libs_to_search_path
+    _add_openvino_libs_to_search_path()
+except ImportError:
+    pass
+
+# API 2.0
+try:
+    # Import all public modules
+    from openvino import runtime as runtime
+    from openvino import frontend as frontend
+    from openvino import helpers as helpers
+    from openvino import preprocess as preprocess
+    from openvino import utils as utils
+    from openvino.runtime import properties as properties
```

Comment:
Experiment showed that there is no need to have these imports: tried to remove this block and imported from runtime, frontend, all worked fine. I suppose we should have a test that checks the imports work and we are done.

Reply:
Yes, it works. 

I suppose here author wants to achieve fast access without boilerplate imports on top of file

Reply:
It works but as @ilya-lavrenov mention, my goal is to provide fast interface like this (if anyone would like to use it):
```python
import openvino as ov
ov.runtime.Something(...)
```
instead of:
```python
import openvino as ov
# some code...
import openvino.runtime as ov_runtime
ov_runtime.Something(...)
```

Reply:
Just tried
```
import openvino as ov
ov.frontend.pytorch.Decoder
```
and it worked even without those imports. Tried stuff from runtime as well. Don't know how it work though.

Reply:
> Don't know how it works though

I also don't know, because `openvino/frontend/__init__.py` does not import `pytorch`
@jiwaszki any idea?